### ISSUE TITLE
[TW-241] new column format; recur.short

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#284]
_David Patrick on 2013-09-08T19:10:16Z says:_ 

 The recur column already has recur.indicator, but in the same single character space, recurrence periods could be represented wit d, w, m, y.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2013-09-08T19:10:16Z
Modified: 2014-09-06T19:34:36Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2014-02-09T18:49:31Z says:_ 

 This should be fixed in 2.4.0 automatically when recurrence periods are rendered using the ISO-8601 minimal format, so "1 month" will be saved as "P1M".  That's as short as its going to get.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2014-09-06T19:34:36Z says:_ 

 Instead of a new 'recur.short' format, the default format for all duration attributes (recur, and any UDA) is now the compact ISO format, as discussed above.
